htmresearch/frameworks/nlp/classify_htm.py

```diff
@@ -106,7 +106,7 @@ def inferToken(self, token, reset=0, returnDetailedResults=False,
     sensor = self.sensorRegion.getSelf()
     sensor.addDataToQueue(token,
                           categoryList=[None],
-                          sequenceId=-1, reset=0)
+                          sequenceId=-1, reset=reset)
```

Comment:
No because the ClassificationModel may do something on reset.  However, I think we can remove the specific alls to each Region's reset in ClassificationNetworkAPI. I'll do that (after testing)